Dshop/apps/users/tests/test_api_password_change.py

```diff
@@ -0,0 +1,124 @@
+import json
+
+import pytest
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.authtoken.models import Token
+
+User = get_user_model()
+
+
+@pytest.fixture
+def password_change_url():
+    return reverse('api-password-change')
+
+
+@pytest.fixture

```

Comment:
repeats in other test, please remove or move to conftests.py 
Please also check other fixtures and move to conftest.py if repeated.